test/SdkTests/TestConfig.xml

```diff
@@ -0,0 +1,145 @@
+<Tests>
+  <TestList Name="SdkIntegrationTests">
```

Comment:
It runs the tests in the `SdkIntegrationTests` test list.  That's specified in the command line args to the test runner (`-testList SdkIntegrationTests`) in SdkTests.csproj.  It will also skip any of the tests that are marked as skipped.